ui_src/src/components/selectThroughput/index.js

```diff
@@ -35,17 +35,22 @@ const selectThroughput = ({ options, onChange, value }) => {
                 onChange={handleChange}
                 placement="bottomRight"
                 popupClassName="select-throughput-options"
+                listHeight={210}
             >
-                {options?.map((component) => {
-                    return (
-                        <Option key={component?.name} value={component?.name}>
-                            <div className="throughput-details">
-                                <img src={ComponentIcon} alt="ComponentIcon" height="18px" />
-                                <p className="throughput-name">{component?.name}</p>
-                            </div>
-                        </Option>
-                    );
-                })}
+                {options
```

Comment:
The sorting function should be applied to the data before it is inserted into the DOM

Reply:
@avrhamNeeman Changed the sort from lexicographic sort to numerical based on component number